.env.e2e.example

```diff
@@ -0,0 +1,33 @@
+#haitatonurls
+TA_HAITATON_TESTING="http://localhost:3001/fi"
+TA_HAIT_TEST_HANKESALKKU="http://localhost:3001/fi/hankesalkku/"
+#alluurls
+TA_ALLU_PW=
+TA_ALLU_TESTING=
+# asiakas
+TA_SUOMIFI_LOGIN=
```

Comment:
Maybe there should be some mention somewhere that it is best to use the user on the Confluence page and NOT the user usually used in manual testing? Because then tests might fail. I did that mistake the first time I tried this but I should have known better.

Reply:
I added a paragraph to the E2E Confluence page under 'Ympäristomuuttujat'.